03-Raster-Calculations-In-R.Rmd

```diff
@@ -133,6 +145,23 @@ plot(DSM_HARV,
      main="Digital Surface Model (Elevation)\n Harvard Forest")
 ```
 
+Notice that the elevation range for these two maps (DSM and DTM) is different. 
+Why?  Should we be worried about this?  
+
+##Two Ways to Calculate
+We can calculate the difference between two rasters in two different ways:
+
+1. by directly subtracting the two rasters in `R` using raster math
+
+or if our rasters are large and/or the calculations we are performing are 
+complex:
+
+2. using the `overlay()` function.
+
+##Raster Math & Canopy Height Models
+First, we'll learn the intuitive process of subtracting layers to create and
```

Comment:
We can perform raster calculations by simply subtracting (or adding, multiplying etc) two rasters. In the geospatial world, we often...